test/e2e-frontend/members.test.js

```diff
@@ -116,6 +116,26 @@ describe('Front-end members behaviour', function () {
                 .expect(400);
         });
 
+        it('should error for fetching member newsletters with missing uuid', async function () {
+            await request.get('/members/api/member/newsletters')
+                .expect(400);
+        });
+
+        it('should error for fetching member newsletters with invalid uuid', async function () {
+            await request.get('/members/api/member/newsletters?uuid=abc')
+                .expect(400);
+        });
+
+        it('should error for updating member newsletters with missing uuid', async function () {
+            await request.put('/members/api/member/newsletters')
+                .expect(400);
+        });
+
+        it('should error for updating member newsletters with invalid uuid', async function () {
+            await request.put('/members/api/member/newsletters?uuid=abc')
+                .expect(400);
+        });
```

Comment:
We are missing tests for the success cases. IMO line 59 needs to change from

`await testUtils.initFixtures('members');` 

to

`await testUtils.initFixtures('newsletters', 'members:newsletters');` 

and we should be testing that we get the data we expect?

Reply:
@ErisDS Yea totally, updated now, thanks 🙏